coalib/output/ConsoleInteraction.py

```diff
@@ -599,12 +623,21 @@ def print_actions(console_printer, section, actions, failed_actions):
                             values for the action. If the user did
                             choose to do nothing, return (None, None).
     """
-    choice = choose_action(console_printer, actions)
-
-    if choice == 0:
-        return None, None
-
-    return get_action_info(section, actions[choice - 1], failed_actions)
+    if section.name == 'cli':
```

Comment:
The choice is 1st anyway. I do this because when I do the next line, if the `reprompt` value doesn't exists it will return by default an empty string. Also this was the same as always, i just changed the next line with `StringConverter(section.get('reprompt').value)).__bool__():`